docs/rst/use_cases/vulcanexus_cloud.rst

```diff
@@ -0,0 +1,150 @@
+.. include:: ../exports/alias.include
+
+.. _vulcanexus_cloud:
+
+################
+Vulcanexus Cloud
+################
+
+*Vulcanexus* is an extended *ROS 2* distribution provided by *eProsima* which includes additional tools for an improved
+user and development experience, such as *Fast-DDS-Monitor* for monitoring the health of DDS networks, or *micro-ROS*,
+a framework aimed at optimizing and extending *ROS 2* toolkit to microcontroller-based devices.
+
+Apart from plain LAN-to-LAN communication, Cloud environments such as container-oriented platforms have also been
+present throughout the |ddsrouter| design phase. In this walk-through example, we will set up both a *Kubernetes*
+(|k8s|) network and local environment in order to establish communication between a pair of ROS nodes, one sending
+messages from a LAN (talker) and another one (listener) receiving them in the Cloud. This will be accomplished by having
+a |ddsrouter| instance at each side of the communication.
+
+.. figure:: /rst/figures/ddsrouter_overview_wan.png
+
+Local setup
+===========
+The local instance of |ddsrouter| (local router) only requires to have a :ref:`Simple Participant <user_manual_participants_simple>`,
+and a :ref:`WAN Participant <user_manual_participants_wan>` that will play the client role in the discovery process of
+remote participants (see `Discovery Server <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html>`_).
+
+After having acknowledged each other’s existence through `Simple DDS discovery mechanism <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html>`_
+(mutlicast communcation), the local participant will start receiving messages published by the ROS 2 talker node, and
+will then forward them to the WAN participant. Following, these messages will be sent to another participant hosted on a
+|k8s| cluster to which it connects via WAN communication over UDP/IP.
+
+Following is a representation of the above-described scenario:
+
+.. figure:: /rst/figures/vulcanexus_local.png
```

Comment:
Not sure about term `UDP Tunnel`. Please, ask to more experienced people.

Reply:
Not sure about this one. Stream, connection?

Reply:
I went for UDP data transmission.